lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -416,109 +405,81 @@ sub ws_send {
 
 sub settings_hash {
     my ($self) = @_;
-
-    if (!defined($self->{_settings})) {
-        $self->{_settings} = {};
-        for my $var ($self->settings->all()) {
-            if (defined $self->{_settings}->{$var->key}) {
-                # handle multi-value WORKER_CLASS
-                $self->{_settings}->{$var->key} .= ',' . $var->value;
-            }
-            else {
-                $self->{_settings}->{$var->key} = $var->value;
-            }
+    return $self->{_settings} if defined $self->{_settings};
+    $self->{_settings} = {};
+    for ($self->settings->all()) {
```

Comment:
It would be good to give the loop object a name when changing this code anyways.

Reply:
Actually I deleted the explicit loop variable because it was called just `$var` which I don't find clearer than `$_`. I prefer `$_` unless there is a good reason we should have a well named loop variable. We could use `$setting` but I think here it's clear enough that each item is a "setting", if you agree.